amethyst_rendy/src/pass/base_3d.rs

```diff
@@ -666,8 +628,14 @@ impl<B: Backend, T: Base3DPassDef<B>> RenderGroup<B, Resources> for DrawBase3DTr
                         if let Some(mesh) =
                             B::unwrap_mesh(unsafe { mesh_storage.get_by_id_unchecked(*mesh) })
                         {
-                            mesh.bind_and_draw(0, &self.vertex_format_base, range.clone(), encoder)
-                                .unwrap();
+                            if let Err(error) = mesh.bind_and_draw(
+                                0,
+                                &self.vertex_format_base,
+                                range.clone(),
+                                encoder,
+                            ) {
+                                log::warn!("Trying to draw a mesh that lacks {:?} vertex attributes. Pass {} requires attributes {:?}.", error.not_found.attributes, T::NAME, T::base_format());
```

Comment:
rustfmt can't handle, haha. I think if you split the format string using `\` it will be able to.
same below

Reply:
I think just dropping the macro parameters to separate lines should do the trick as well :)